applications/plugins/CGALPlugin/src/CGALPlugin/CylinderMesh.inl

```diff
@@ -495,6 +495,8 @@ void CylinderMesh<DataTypes>::orientate()
 template <class DataTypes>
 void CylinderMesh<DataTypes>::draw(const sofa::core::visual::VisualParams*)
 {
+#ifndef SOFA_NO_OPENGL
```

Comment:
This is not the correct way to fix that. You should use the drawTools instead of using direct gl code.